docs/02-guides/override-a-stylesheet.mdx

```diff
@@ -8,9 +8,205 @@ sidebar_position: 2
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+import Figure from "@site/src/components/Figure";
 
-See
-https://developers.front-commerce.com/docs/2.x/essentials/adapt-theme-to-your-brand
+It is not something that you can do in just a few minutes because your identity
+shines in the details. However, you can do all the heavy lifting pretty quickly
+by customizing what is called **Design Tokens**.
+
+And it's a great way to familiarize yourself with Front-Commerce!
+
+## Get your Design Tokens
+
+This concept is often closely related to Design Systems. Basically Design Tokens
+are the _core styles_ of your design: colors, font families, font sizes,
+borders, shadows, etc. Together they are what make your brand unique.
+
+There even exist tools that extract those tokens from existing websites. In this
+example, we will use [CSS Stats](https://cssstats.com/) to extract Design Tokens
+from [Smashing Magazine](https://www.smashingmagazine.com/) and use them later.
+
+<Figure caption="Here is an example with the awesome Smashing Magazine website">
+
+![CSS Stats lists all the 62 colors and 59 unique background colors of Smashing Magazine](./assets/smashingmagazine-cssstats.png)
+
+</Figure>
+
+:::info
+
+If you want to learn more about it, you can have a look at
+[Design tokens for dummies](https://uxdesign.cc/design-tokens-for-dummies-8acebf010d71)
```

Comment:
**note:** in the linked article, an update suggest that the updated version of the article is at this link instead:
https://specifyapp.com/blog/introduction-to-design-tokens

Reply:
updated !